packages/vite/src/node/plugins/css.ts

```diff
@@ -1211,8 +1213,8 @@ async function minifyCSS(css: string, config: ResolvedConfig) {
   try {
     const { code, warnings } = await transform(css, {
       loader: 'css',
-      minify: true,
-      target: config.build.cssTarget || undefined
+      target: config.build.cssTarget || undefined,
+      ...resolveEsbuildMinifyOptions(config.esbuild || {})
```

Comment:
It might be nicer if we could pre-compute this somewhere, but I can't find a good spot to put without making the flow a bit awkward. Perhaps it's fine with V8